.github/workflows/cd.yml

```
@@ -0,0 +1,32 @@
+name: CD — Deploy to VPS
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)